src/test/java/org/xstefank/check/SkipCheckTest.java

```diff
@@ -0,0 +1,91 @@
+package org.xstefank.check;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.xstefank.TestUtils;
+import org.xstefank.api.GitHubAPI;
+import org.xstefank.model.yaml.FormatConfig;
+import org.xstefank.model.yaml.SkipPatterns;
+import java.util.regex.Pattern;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(GitHubAPI.class)
+public class SkipCheckTest {
+
+    private static FormatConfig testFormatConfig = new FormatConfig();
```

Comment:
the instance you create here is actually overriden in every method that uses this field by the call to `TestUtils.setUpFormatConfig` so it's not necessary to be created